sla-template-descriptor/sla-template-schema.yml

```diff
@@ -0,0 +1,292 @@
+---
+"$id": http://example.com/example.json
+type: object
+definitions: {}
+"$schema": http://json-schema.org/draft-06/schema#
```

Comment:
Better to stick to draft-04 to be consistent with other schemas. Not sure if the validator can check draft-06 schemas